adoc/admin-troubleshooting.adoc

```diff
@@ -57,6 +98,25 @@ For example, if bootstraps the first master node of the cluster fails, re-run th
 skuba node bootstrap --user sles --sudo --target <IP/FQDN> <NODE NAME> -v=1
 ----
 
+However, if the `bootstrap` (or `join`) procedure fails at the last final steps, re-running it might _not_ help. To verify
```

Comment:
I think we can reduce this case to `join` only. Bootstrap is safe to be retried, as we'll just `kubeadm reset`.

Also, regarding join, it would be good to know under what conditions this can fail and if we can do something about it (if skuba detects that the join was partially made, as part of the "rollback" operation it could delete the node from the cluster if it happened to register, and possibly perform other tasks).

Reply:
To be on the safe side, I would propose to leave it as it is for now. Running `kubeadm` commands outside of `skuba` sounds scary to me, so I would avoid telling the customers to interact with kubeadm but use only skuba as their interface.

But, you are right. This is not a proper fix. Thus, I would propose I could create another GH where we could focus on a better solution, addressing the root problem and involve QA as well.

Reply:
Oh no no, I mean that we are [literally doing a `kubeadm reset` with skuba](https://github.com/SUSE/skuba/blob/f8534b1bc369335870f948715ee7f0a446df0b2d/pkg/skuba/actions/node/bootstrap/bootstrap.go#L137), so the operation can be retried.

Reply:
I see, yes that makes sense. I narrowed down the scope only to `join`.